docs/user_guide.md

```diff
@@ -248,3 +249,75 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
 
 When an image is signed, it has a tick shown in UI; otherwise, a cross sign(X) is displayed instead.  
 ![browse project](img/content_trust.png)
+
+### Vulnerability scan via Clair 
+**CAUTION: Clair is an optional installing component, please make sure you have already enable it in your Harbor before you go through this part!**
+
+Static analysis of vulnerabilities is provided through open source project [Clair](https://github.com/coreos/clair). You can trigger scan action on any images you want to as well as scan all the images in Harbor at one time. Additionally, you can also set a policy to scan all the images at a specified time everyday.
+
+**Vulnerability metadata** 
+
+Clair depends on the vulnerability metadata to complete the analysis process. After the first initial installation, Clair will automatically start to update the metadata database from different vulnerability repositories. The updating process might be take a while based on the data size and network situations etc.. If the database is not fully ready, there will be a warning tips at the footer of the repositories datagrid view.

```

Comment:
The updating process might be take a while based on the data size and network situations etc.. If the database is not fully ready, there will be a warning tips at the footer of the repositories datagrid view.
==>
The updating process may take a while based on the data size and network connection. If the database is not fully populated, there is a warning message at the footer of the repository datagrid view.